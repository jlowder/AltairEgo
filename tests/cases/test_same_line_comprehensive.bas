10 REM Test various same-line FOR NEXT scenarios
20 FOR I=1 TO 3: PRINT "A"; I: NEXT I: PRINT " Done with A"
30 FOR J=5 TO 7: FOR K=1 TO 2: PRINT J; K: NEXT K: NEXT J
40 FOR X=10 TO 12 STEP 1: PRINT "X="; X: NEXT X: PRINT "Finished"
50 END
