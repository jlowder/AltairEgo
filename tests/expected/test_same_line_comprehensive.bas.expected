Altair Ego: Emulating Altair BASIC 32K Rev. 3.2
OK
OK
A 1
A 2
A 3
 Done with A
 5  1
 5  2
 6  1
 6  2
 7  1
 7  2
X= 10
X= 11
X= 12
Finished
OK
